doc/examples/example_logger.py

```diff
@@ -4,19 +4,38 @@
 
 """
 
-import pandas as pd
+import logging
 
+import pandas as pd
 import pastas as ps
 
-# using the default logger, we will see some information in the console:
+# When the script running pastas (this script) does not initialize a logger,
+# only warnings and errors are printed to the console
 obs = pd.read_csv("data/B58C0698001.csv", parse_dates=[0], index_col=0).squeeze()
 
 ts = ps.Model(obs)
 
-# when we set the level to WARNING we do not see any output anymore
-ps.set_log_level("WARNING")
-# which is just a wrapper around
-ps.utils.set_console_handler(ps.logger, level="ERROR")
+# when we set the level to ERROR we do not see the warning anymore
+ps.set_log_level("ERROR")
+
+ts = ps.Model(obs)
+
+# when we set the log-level to "INFO", info-messages are still not printed
+ps.set_log_level("INFO")
```

Comment:
Should we not deprecate this method then?

Reply:
The set_log_level-method is useful, as it sets the log-level of Pastas. The messages are still not printed, though, as we have not initialized a logger yet (which is what we do in the next line).